accesslog/schema/access_log_record_test.go

```diff
@@ -314,6 +314,27 @@ var _ = Describe("AccessLogRecord", func() {
 				Eventually(r).Should(Say(`x_cf_routererror:"-"`))
 			})
 		})
+
+		Context("when extra_fields is set to [local_address]", func() {
```

Comment:
Can you add an test for...
* when extra_fields does not contain local_address, then the local address is not tested.
* when extra_fields contains "meow" it ... does not error? logs? idk does something reasonable?